gitcash/src/main.rs

```diff
@@ -116,6 +119,26 @@ pub fn main() -> anyhow::Result<()> {
     // Parse args
     let args = Args::parse();
 
+    if args.command == Command::ConnectPointOfSale {
+        if args.config.exists() {
+            bail!(format!(
+                "❌ Config File {:?} already exists!",
+                args.config.display()
+            ));
+        }
+
+        let example = include_str!("../../config.toml.example");
+
+        println!("💰 Wrote {:?} 🏦\n", args.config.display());
+
+        return write(&args.config, example).with_context(|| {
+            format!(
+                "unable to write example config to {:?}",
+                args.config.display()
+            )
+        });
```

Comment:
At this point, the config wasn't yet written, so the log "Wrote" is not accurate 🙂 Additionally, I find the "return <value>" a bit confusing.

How about this?

```suggestion
        write(&args.config, example).with_context(|| {
            format!(
                "unable to write example config to {:?}",
                args.config.display()
            )
        })?;
        println!("💰 Wrote config to {:?} 🏦\n", args.config.display());
        return;
```